data/sql/updates/pending_db_world/Scarlet_miners_and_Cars_Fix.sql

```diff
@@ -0,0 +1,253 @@
+(12889800, 12, 2282.96, -5948.26, 60.8342, NULL, 0, 0, 0, 100, 0),
+(12889800, 13, 2278.88, -5960.01, 54.3672, NULL, 0, 0, 0, 100, 0),
+(12889800, 14, 2275.86, -5972.99, 50.1883, NULL, 0, 0, 0, 100, 0),
+(12889800, 15, 2266.45, -5975.92, 45.7126, NULL, 0, 0, 0, 100, 0),
+(12889800, 16, 2257.05, -5977.68, 40.1854, NULL, 0, 0, 0, 100, 0),
+(12889800, 17, 2248.05, -5979.76, 35.7801, NULL, 0, 0, 0, 100, 0),
+(12889800, 18, 2240.11, -5989.08, 30.5966, NULL, 0, 0, 0, 100, 0),
+(12889800, 19, 2232.67, -6001.29, 22.5347, NULL, 0, 0, 0, 100, 0),
+(12889800, 20, 2227.55, -6009.1, 15.5793, NULL, 0, 0, 0, 100, 0),
+(12889800, 21, 2222.56, -6017.32, 10.2762, NULL, 0, 0, 0, 100, 0),
+(12889800, 22, 2219.65, -6035.31, 7.07337, NULL, 0, 0, 0, 100, 0),
+(12889800, 23, 2217.22, -6052.6, 6.29405, NULL, 0, 0, 0, 100, 0),
+(12889800, 24, 2226.42, -6062.52, 6.18018, NULL, 0, 0, 0, 100, 0),
+(12889800, 25, 2237.33, -6073.54, 5.80553, NULL, 0, 0, 0, 100, 0),
+(12889800, 26, 2247.76, -6083.73, 6.15304, NULL, 0, 0, 0, 100, 0),
+(12889800, 27, 2254.38, -6094.76, 6.24679, NULL, 0, 0, 0, 100, 0);
+
+
+-- Activate SmartAI to Scarlet Miners and Mine Cars
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` = 28822);
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE (`entry` = 28821);
+
+
+-- SMARTAI
+
+-- Mine Cars
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 28821;
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = 28821);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(28821, 0, 0, 0, 11, 0, 100, 0, 0, 0, 0, 0, 0, 0, 29, 1, 180, 0, 0, 0, 0, 23, 0, 0, 0, 0, 0, 0, 0, 0, 'Mine Car - On Respawn - Start Follow Owner Or Summoner');
+
+
+-- Scarlet Miners
+
+-- Miners inside the Mine
+
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND `entryorguid` = -128895);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `event_param6`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(-128895, 0, 0, 1, 11, 0, 100, 0, 0, 0, 0, 0, 0, 0, 12, 28821, 8, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 'Scarlet Miner - On Respawn - Summon Creature \'Mine Car\''),
```

Comment:
can cou please leave a comment like "has guid specific SAI" in the creature table for these guids?
this helps keeping an overview about entries with guid-specific SAI :)

Reply:
Done.